### PR TITLE
logging: add user information to sentry

### PR DESCRIPTION
**Pull request by rerowep**
* Closes: #2734.

### CODE REVIEW

File: rero_ils/config.py [hunk 1]
```diff
@@ -3455,6 +3455,11 @@ def _(x):
 #: Sentry: use celery or not
 LOGGING_SENTRY_CELERY = True
 
+LOGGING_SENTRY_INIT_KWARGS = {
+    # Instruct Sentry to send user data attached to the event
+    "send_default_pii": True,
```
Comment (PascalRepond) [inline rero_ils/config.py:3460]: pii?

File: rero_ils/modules/utils.py [hunk 2]
```diff
@@ -1091,6 +1092,7 @@ def set_user_name(sender, user):
 
     user_name = None
     remove_user_name(sender, user)
+    set_user_data = {"id": user.id, "email": user.email}
```
Comment (PascalRepond) [inline rero_ils/modules/utils.py:1095]: User email is not useful and might be a security concern to put in the logs.

Comment (PascalRepond) [reply rero_ils/modules/utils.py:1095]: More useful information to debug and reproduce would be : `patron.pid` `patron.patron.barcode[0]` and the patron's organisation.

File: rero_ils/config.py (hunk 1 above)
Comment (rerowep) [reply rero_ils/config.py:3460]: See sentry sdk

**Review by PascalRepond — APPROVED** [review]

File: rero_ils/modules/utils.py [hunk 3]
```diff
@@ -1085,22 +1084,35 @@ def close(self):
         self.__del__()
 
 
+def user_formatted_name(user):
```
Comment (jma) [inline rero_ils/modules/utils.py:1087]: You can use the Patron.formatted_name method.

**Review by jma — APPROVED** [review]

**Pull request merged**